packages/manager/modules/web-paas/src/add/add.controller.js

```diff
@@ -147,12 +138,14 @@ export default class {
   }
 
   onPlatformOrderSuccess(checkout) {
-    if (checkout.prices && checkout.prices.withTax.value > 0) {
+    if (checkout && checkout.prices && checkout.prices.withTax.value > 0) {
       this.$window.open(checkout.url, '_blank', 'noopener');
     }
     this.Alerter.success(
       this.$translate.instant('web_paas_add_project_success', {
-        orderURL: this.getOrdersURL(checkout.orderId),
+        orderURL: checkout
+          ? this.getOrdersURL(checkout.orderId)
+          : this.getOrdersURL(),
```

Comment:
```suggestion
        orderURL: this.getOrdersURL(checkout)
```
check for checkout is empty in getOrderUrl method